ui/entry-failureviewer.js

```diff
@@ -19,8 +19,9 @@ require('angular-resource');
 require('angular-cookies');
 require('angular-sanitize');
 require('angular-local-storage');
+require('popper.js/dist/umd/popper');
 require('bootstrap/dist/js/bootstrap');
```

Comment:
Let's switch the bootstrap entry to use `import` and the format recommended here:
https://getbootstrap.com/docs/4.0/getting-started/webpack/#importing-javascript

Also, that page doesn't say we have to also `require('popper.js/dist/umd/popper')`, were things not working without that?

Reply:
Yeah, you're right.  I took it out and it still worked.  I was hitting so very many problems earlier on that I think I'd tried that and hadn't taken it out yet...  :)  Thanks for catching that.